osx/ka-lite-remover.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+
+# Notes: 
+#    * This script must be run as root.
+#    * The files that will be removed, will be displayed on the console log.
+#
+# What does this script do?
+#    1. Unset environment variable: KALITE_PYTHON.
```

Comment:
Yup It is, When we remove the .plist file we unset the  env variable too
So that we ensure that there is no `KALITE_PYTHON ` remnant in the env.